CMakeLists.txt

```diff
@@ -385,6 +385,10 @@ include_directories("${TF_INC_DIR}")
 endif() # USE_TF
 
 # OpenCV
+find_package(OpenCV 3 QUIET)
+if (NOT OpenCV_FOUND)
+  find_package(OpenCV 2 REQUIRED)
+endif()
```

Comment:
There seems to be yet another find_package for opencv below this line, do we need it ?

Reply:
no we dont, fixed